ArraysAndStrings/isUnique/solution1c.js

```diff
@@ -0,0 +1,17 @@
+//Implement am algorithm to determine if a string has all unique characters
```

Comment:
so you want only new solutions....?

Reply:
I've met interviewers that absolutely hate it when you use a hash map and always set the value to 1 or true.  So this solution shouldn't be merged. Just use the Set instead and your interviewers will love it.

Reply:
@yeomanse Do they have similar dislike if you use a hash if you are using it to get the total count? That is, you set it to one to initialize it, but it doesn't stay there? Or is it only when it is that you could use a Set but you don't?

Reply:
Yes that is totally correct, for example your isPermutation is a perfect use of a hash map.  They just don't like it when you use a hash map when you could use a set because they think it means either you don't know enough about JS to know there is a set or you don't know the difference between the two.  When I give an interview and I see someone use a hash map when they could have used a set, I will give them a chance for redemption and ask if a set would be better.  Other interviewers are not that kind and will silently mark a point off in their head.

Reply:
I'm really confused...is this being submitted somewhere? I thought we were just inputting answers?

Reply:
also can you give me an example where you've experienced the interviewer hating hashmap?

Reply:
Haha its just a place where you guys can look at good answers to these common interview questions.  Hash maps are the best, all interviewers love hash maps EXCEPT when you should use a Set instead.  So we should prevent merging solutions you wouldn't want to use in an interview.  
I've had multiple fellow interviewers tell me they have a hard time seeing past somebody using a hash map when they should have used a set.